cmf-cli/Commands/init/InitCommand.cs

```diff
@@ -174,7 +182,44 @@ public override void Configure(Command cmd)
                 aliases: new[] { "--nugetRegistryPassword" },
                 description: "NuGet registry password"
             ));
-           
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appId" },
+                description: "Use only if repository type is App. Application identifier"
+            )
+            { IsRequired = false });
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appName" },
+                description: "Use only if repository type is App. Application name"
+            ) { IsRequired = false });
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appAuthor" },
+                description: "Use only if repository type is App. Application author"
+            )
+            { IsRequired = false });
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appDescription" },
+                description: "Use only if repository type is App. Application description"
+            ) { IsRequired = false });
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appTargetFramework" },
+                description: "Use only if repository type is App. Application target framework"
+            ) { IsRequired = false });
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appLicensedApplication" },
+                description: "Use only if repository type is App. Licensed application"
+            ) { IsRequired = false });
+
+            cmd.AddOption(new Option<string>(
+                aliases: new[] { "--appIcon" },
+                description: "Use only if repository type is App. Application icon"
+            ) { IsRequired = false });
+

```

Comment:
Since it is an init file it should always be the default start, right?
Either 0.1.0 or 1.0.0

I can instead allow a version parameter with a default value

Reply:
Yes, I understand your perspective and I subscribe it. But shouldn't we allow to define the AppVersion? Maybe @m-s- can help here